app/views/accounts/_og.html.haml

```diff
@@ -0,0 +1,8 @@
+%meta{ property: 'og:site_name', content: site_title }/
+%meta{ property: 'og:type', content: defined?(type) ? type : 'website' }/
```

Comment:
It looks like you added this back into the show view, but did not remove it from here...?